### PR TITLE
[presentation-api] Add a test for starting presentations on distinct displays

### PR DESCRIPTION
**Pull request by tomoyukilabs**
This PR addresses #11233 by adding a test for starting presentations on two distinct displays.

### CODE REVIEW

**Comment by tomoyukilabs** [comment]
ping @tidoust @louaybassbouss

**Review by tidoust — APPROVED** [review]
That looks good to me. The test goes beyond merely testing starting presentation on distinct displays (e.g. it also tests `reconnect`), but that seems ok and useful. We can split the test later on if that seems needed.

**Comment by tomoyukilabs** [comment]
@tidoust I agree because the reconnecting test looks long enough. Now I've committed the change to split the test. PTAL.

**Review by tidoust — APPROVED** [review]
Looks good, thanks!

File: presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    connection2.close();
+    await eventWatcher2.wait_for('close');
+    assert_equals(connection2.state, 'closed', 'The presentation connection is successfully closed.');
+
+    const c1 = await request1.reconnect(connection1.id);
+    assert_equals(c1, connection1, 'The promise is resolved with the existing presentation connection.');
+
+    // Reconnecting a presentation via a different presentation request with the same presentation
+    // URLs will succeed
+    const c2 = await request1.reconnect(connection2.id);
```
Comment (mfoltzgoogle) [inline presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html]: Note that it should be possible to call request2.reconnect here and above, and achieve the same result.

File: presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    const presentBtn = document.getElementById("presentBtn");
+
+    const request1 = new PresentationRequest(presentationUrls);
+    const request2 = new PresentationRequest(presentationUrls);
+    const clickWatcher = new EventWatcher(t, presentBtn, 'click');
+    let connection1, connection2, eventWatcher1, eventWatcher2, timer;
+
+    t.add_cleanup(() => {
+      [
+        { connection: connection1, request: request1 },
```
Comment (mfoltzgoogle) [inline presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html]: It looks like the request values are not used in the .forEach below.

Comment (tomoyukilabs) [reply presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html]: The `request` value is used to reconnect a presentation in the `closed` state and then terminate it. Please see the lines 37-38:

```js
          if (p.connection.state == 'closed')
             p.request.reconnect(p.connection.id);
```

File: presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html (hunk 1 above)
Comment (tomoyukilabs) [reply presentation-api/controlling-ua/reconnectToMultiplePresentations_success-manual.https.html]: I agree. I'll submit another PR to address this. Thanks!

**Pull request merged**